docs/node/migrations/zero-downtime-migrations/migration-proposal.mdx

```diff
@@ -0,0 +1,106 @@
+
+```
+// MigrationResolution is the definition for the network migration vote type in Kwil's
+// voting system.
+var MigrationResolution = resolutions.ResolutionConfig{
+	ConfirmationThreshold: big.NewRat(2, 3),
+	ExpirationPeriod:      100800, // 1 week
+	ResolveFunc: func(ctx context.Context, app *common.App, resolution *resolutions.Resolution, block *common.BlockContext) error {
+		return startMigration(ctx, app, resolution, block)
+	},
+}
+```
+
+## Submit Migration Proposal
+
+A validator operator can submit a migration proposal to upgrade the network using the `kwil-admin migrate propose` command as shown below. Refer to the [kwil-admin migrate](/docs/ref/kwil-admin/migration/propose) documentation for more details.
+
+```bash
+kwil-admin migrate propose --activation-period <activationPeriod> --duration <migrationDuration> --chain-id <newChainID> --rpcserver /tmp/kwil.sock
+```
+
+:::note
+An off-chain communication is required among the validator operators to decide on the details of the migration proposal. One of the validators should submit the proposal to the network for approval. Only one proposal can be applied to the network during the lifetime of the network. If multiple validators submit the same proposal, the network will only consider the first proposal submitted.
+:::
+
+To check if the proposal is submitted successfully, we can get the list of pending migration proposals as shown below:
+
+```bash
+kwil-admin migration list --rpcserver /tmp/kwild.socket
+```
+This list will contain the proposal ID which is used by the validators to vote on the proposal.
+
+
+## Approve Migration Proposal
+
+To start the migration, the migration proposal needs the approval from at least a supermajority of the validators. The validators can submit their vote approval on the migration proposal using the below command.
+
+```bash
+kwil-admin migration approve <proposal-id>
+```
+
+Operators can check the status of the proposal and the votes information as shown below:
+
+```bash
+kwil-admin migration status <proposal-id>
+```
+
+:::note
+The migration proposal will get expired if the supermajority of the validators do not approve the proposal within `100800` blocks (roughly 7 days considering the block rate of 6sec). The proposal can be resubmitted after the expiration.
+:::
+
+### Existing Network Moves Into Migration Mode
+
+Once the proposal is approved, migration is set to start after the specified activation period from the approval block. At the start of the migration, all the existing nodes will generate a snapshot of the network state at the activation height and a genesis file to be used by the new network. The snapshots and genesis files generated at the start of the migration can be downloaded as shown below:
+
+```bash
+kwil-admin migration genesis-state --output-dir </path/to/root-dir>
```

Comment:
Where is it expected that this command should be run?  I just ran it on the old node itself and manually copied the files over to the new node.  But if the admin service is accessible from the new node (config out of scope here) it can be run on the new node directly.

Would be best to comment on this.